lib/plugins/audit.js

```diff
@@ -25,6 +24,7 @@ var HttpError = require('restify-errors').HttpError;
 function auditLogger(options) {
```

Comment:
Add docs about the server object. You'll also want to optionally assert that the server parameter is an object.